docker-compose.yml

```diff
@@ -0,0 +1,39 @@
+version: '3.4'
+
+services:
+  db:
+    image: postgres:11-alpine
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=password
+    volumes:
+      - db-data:/var/lib/postgresql/data
+
+  app: &app_base
+    image: rails_base_api
+    environment:
+      - DB_HOST
```

Comment:
Just asking: why we can't use `DATABASE_URL`? As I know, this is build-in feature of Rails. If we will use it, we can reduce `database.yml` file.

Reply:
Yes, we could. How do you want to define it and where?

Reply:
Solved in fab14e9baa22f08110945cf995d661fd1b2f84c8